python/ray/tune/integration/kubernetes.py

```diff
@@ -0,0 +1,143 @@
+import kubernetes
+import subprocess
+
+from ray import services, logger
+from ray.autoscaler.command_runner import KubernetesCommandRunner
+from ray.tune.syncer import NodeSyncer
+from ray.tune.sync_client import SyncClient
+
+
+def NamespacedKubernetesSyncer(namespace):
+    """
+    Wrapper to return a ``KubernetesSyncer`` for a specific
+    Kubernetes namespace.
+
+    Args:
+        namespace (str): Kubernetes namespace.
+
+    Returns: A ``KubernetesSyncer`` class to be passed to
+        ``tune.run(sync_to_driver)``.
+
+    Example:
+
+        .. code-block:: python
+
+        from ray.tune.integration.kubernetes import NamespacedKubernetesSyncer
+        tune.run(train,
+                 sync_to_driver=NamespacedKubernetesSyncer("ray"))
+
+    """
+
+    class _NamespacedKubernetesSyncer(KubernetesSyncer):
+        _namespace = namespace
+
+    return _NamespacedKubernetesSyncer
+
+
+class KubernetesSyncer(NodeSyncer):
+    """
+    KubernetesSyncer used for synchronization of checkpoints between
```

Comment:
same line, but no overflow (it will reflect in documentation)


```
class ExampleClass(object):
    """The summary line for a class docstring should fit on one line.